makefile

```diff
@@ -863,7 +863,7 @@ ifeq ($(platform),windows)
 	exe-suffix = .exe
 	rpath =
 
-	lflags = -L$(lib) $(common-lflags) -lws2_32 -liphlpapi -mconsole
+	lflags = -L$(lib) -static-libgcc -static-libstdc++ -Wl,-Bstatic -lstdc++ -lpthread -Wl,-Bdynamic $(common-lflags) -lws2_32 -liphlpapi -mconsole
```

Comment:
I put these options to a wrong line. I should move them to "android cp + windows" case